replica/database.cc

```diff
@@ -801,21 +801,43 @@ future<> database::parse_system_tables(distributed<service::storage_proxy>& prox
 future<>
 database::init_commitlog() {
     if (_commitlog) {
-        return make_ready_future<>();
+        co_return co_await make_ready_future<>();
```

Comment:
Isn't this just `co_return`?

Reply:
That is if you write `co_return make_ready_future<>();`, no? What about `co_return;`?

Reply:
Just `co_return;` works, thanks